libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -533,18 +581,124 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
     decisionHandler(WKNavigationActionPolicyAllow);
 }
 
+- (BOOL) webView:(UIWebView *) webView shouldStartLoadWithRequest:(NSURLRequest *) request navigationType:(UIWebViewNavigationType) navigationType
```

Comment:
Equivalent `UIWebViewDelegate` callback. This is the only method where I couldn't reuse the `WKWebView` code that I had, because `WKWebView` does async and `UIWebView` does sync.